### PR TITLE
STRF-2471 - Multiple Wish Lists

### PR DESCRIPTION
**Pull request by mjschock**
#### What?

* wish list dropdown with the ability to select which wish list to add a product to
* normalizes Wish List copy 
* depends on:
  *  https://github.com/bigcommerce/bigcommerce/pull/22461
  *  https://github.com/bigcommerce/bigcommerce/pull/22732

#### Tickets / Documentation

- [STRF-2471](https://jira.bigcommerce.com/browse/STRF-2471)

#### Screenshots

ping @bigcommerce/storefront-team

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -399,26 +399,28 @@
             "no_items": "The items you've recently looked at on our site will appear below."
         },
         "wishlists": {
-            "heading": "Wish lists",
-            "tag_line": "Fill in the form below to create a wish list. Click the \"Save\" button when you're done.",
-            "new": "New wish List",
-            "you_have_none": "You have no wish lists, add one now.",
-            "empty_wishlist": "Your wish list is empty. When you add items to your wish list they will appear here.",
+            "heading": "Wish Lists",
+            "tag_line": "Fill in the form below to create a Wish List. Click the \"Save\" button when you're done.",
```
Comment (Ubersmake) [inline lang/en.json]: Is "Click the "Save" button when you're done" still accurate? Specifically the "Save." It looks like the button will get "Create New Wish List."

Comment (mjschock) [reply lang/en.json]: good catch, the button in this case is actually "Create Wish List" without the "New".  i've updated the copy, replacing "Save" with "Create Wish List".

**Comment by mjschock** [comment]
@bigcommerce/storefront-team ♻️

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@
 - Fixes issue with image zoom causing scrolling issues on mobile. [#1141](https://github.com/bigcommerce/cornerstone/pull/1141)
 - Fix mis-sized product images. [#1145](https://github.com/bigcommerce/cornerstone/pull/1145)
 - Remove "as low as" feature and add support for price ranges instead[#1143](https://github.com/bigcommerce/cornerstone/pull/1143)
+- Implements Add to any Wish Lists capability. [#1134](https://github.com/bigcommerce/cornerstone/pull/1134)
```
Comment (junedkazi) [inline CHANGELOG.md]: This should be in the draft section.

File: assets/js/theme/wishlist.js [hunk 3]
```diff
@@ -13,6 +13,8 @@ export default class WishList extends PageManager {
         this.options = {
             template: 'account/add-wishlist',
         };
+
+        return this;
```
Comment (junedkazi) [inline assets/js/theme/wishlist.js]: Any particular reason for doing a `return` here ?

File: lang/en.json [hunk 4]
```diff
@@ -399,26 +399,28 @@
             "no_items": "The items you've recently looked at on our site will appear below."
         },
         "wishlists": {
-            "heading": "Wish lists",
-            "tag_line": "Fill in the form below to create a wish list. Click the \"Save\" button when you're done.",
-            "new": "New wish List",
-            "you_have_none": "You have no wish lists, add one now.",
-            "empty_wishlist": "Your wish list is empty. When you add items to your wish list they will appear here.",
+            "heading": "Wish Lists",
+            "tag_line": "Fill in the form below to create a Wish List. Click the \"Create Wish List\" button when you're done.",
```
Comment (junedkazi) [inline lang/en.json]: should we add the work `new` to `Fill in the form below to create a Wish List.` 🍹 .

File: CHANGELOG.md (hunk 2 above)
Comment (mjschock) [reply CHANGELOG.md]: good catch

File: lang/en.json (hunk 4 above)
Comment (mjschock) [reply lang/en.json]: sure

File: assets/js/theme/wishlist.js (hunk 3 above)
Comment (mjschock) [reply assets/js/theme/wishlist.js]: yeah, its so we can chain a call to one of it's methods:  https://github.com/bigcommerce/cornerstone/pull/1134/files#diff-9c4b0efa294566acaa5d14e862d35bc7R20

**Pull request merged**